### PR TITLE
added rest util and manager

### PR DESCRIPTION
**Pull request by festinalente91**

### CODE REVIEW

File: shared/actions/authActions.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-  HTTPS,
-  SERVER_ADDR,
-  ENDPOINT_USER,
-  ENDPOINT_SIGNUP,
-  ENDPOINT_GRANT,
-  ENDPOINT_REFRESH,
-  DEFAULT_HEADERS,
-  getAuthHeaders
+    HTTPS,
+    SERVER_ADDR,
```
Comment (ochanje210) [inline shared/actions/authActions.js]: ditto

File: shared/actions/authActions.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+    ENDPOINT_USER,
```
Comment (ochanje210) [inline shared/actions/authActions.js]: ditto

File: shared/actions/authActions.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+    ENDPOINT_SIGNUP,
```
Comment (ochanje210) [inline shared/actions/authActions.js]: ditto

**Comment by ochanje210** [comment]
lgtm

**Comment by ochanje210** [comment]
I will follow and use your auth func's utilization soon

**Comment by festinalente91** [comment]
awesome!

**Pull request merged**